cheetah/accelerator.py

```diff
@@ -299,6 +307,354 @@ def defining_features(self) -> list[str]:
     def __repr__(self) -> str:
         return f"{self.__class__.__name__}(length={repr(self.length)})"
 
+class SpaceChargeKick(Element):
+    """
+    Simulates space charge effects on a beam.
+    :param length: Length of the element in meters.
```

Comment:
As for now, time discretization is manually implemented through the lenght. If spacechargekick objects are built automatically, it might be interesting to play on this parameter.